### ISSUE TITLE
v8.0.1 quotes commit message

### ISSUE THREAD

**Issue by danez** [conventional-changelog/standard-version#621]
**Describe the bug**
Version 8.0.1 does for some reason quote the commit message.

**Current behavior**

```
commit f8f0ea9bb30d759e091d1cafc034f170dea059f5
Author: Author <author@domain.com>
Date:   Mon Jul 13 11:12:25 2020 +0000
    "chore(release): 8.66.4"
```

**Expected behavior**

```
commit f8f0ea9bb30d759e091d1cafc034f170dea059f5
Author: Author <author@domain.com>
Date:   Mon Jul 13 11:12:25 2020 +0000
    chore(release): 8.66.4
```

**Environment**
- `standard-version` version(s): 8.0.1
- Node/npm version: 12
- OS: linux

**Additional context**
Probably due to: https://github.com/conventional-changelog/standard-version/commit/871201ff8de6091d4b9b2ce9bf8611a1fc2f7836

**Comment by JVMartin** [comment]
![image](https://user-images.githubusercontent.com/4450814/87339924-33ea0980-c4fc-11ea-8728-711a26c5f71d.png)

Confirmed, this is a major issue that results in automation publishing endlessly, since the new commits do not follow semver.

**Comment by JVMartin** [comment]
This is happening to me suddenly even on standard-version `7.1.0`.

**Comment by jbottigliero** [comment]
Thanks for the report - this will be included in `8.0.2` (#622). 

@JVMartin - I just ran the test suite with `standard-version@7.1.0` + `node@12.18.2` and I'm not seeing the quotes in the commit message. If you find you're still having issues could you open a separate report with details on your environment (and maybe your `releaseCommitMessageFormat` configuration if you're not using the default) and I can try to help troubleshoot?
